kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/Channel.java

```diff
@@ -121,30 +91,28 @@ public Channel(final String name, final ChannelType type, final DataType valueTy
      * @param valueType
      *            the value type
      * @param valueScale
-     *            the value used to scale the value, must have the same {@link DataType} as valueOffset
+     *            an optional value used to scale the value, must have the same {@link DataType} as valueOffset
      * @param valueOffset
-     *            the value used as offset of the value, must have the same {@link DataType} as valueScale
+     *            an optional value used as offset of the value, must have the same {@link DataType} as valueScale
      * @param config
      *            the configuration
      * @throws NullPointerException
      *             if any of the arguments is null
      * @throws IllegalArgumentException
      *             if any of the valueScale and valueOffset have different types
      *
-     * @since 2.8
+     * @since 3.0
      */
```

Comment:
we target kura 6, we are allowed to do that right ?

Reply:
We are allowed to do that only for the methods that have been deprecated in a previous release

Reply:
In my opinion we could return 1 as scale and 0 as offset instead of null, if they are not specified. Not specifyng the scale or offset is equivalent to specifing 1 and 0 for them right?

Reply:
i can restore the previous api and deprecate them, i think have null value is better (not much better but more correct)